hazelcast/src/main/java/com/hazelcast/config/ScheduledExecutorConfig.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import static com.hazelcast.util.Preconditions.checkNotNegative;
+
+/**
+ * Created by Thomas Kountis.
+ */
+public class ScheduledExecutorConfig {
+
+    /**
+     * Scheduled of Executor
+     */
+    public static final int DEFAULT_DURABILITY = 1;
+
+    private String name = "default";
+
+    private int durability = DEFAULT_DURABILITY;
+
+    private com.hazelcast.config.ScheduledExecutorConfig.ScheduledExecutorConfigReadOnly readOnly;
+
+    public ScheduledExecutorConfig() {
+    }
+
+    public ScheduledExecutorConfig(String name) {
+        this.name = name;
```

Comment:
Do the arguments need to be checked? Most config have some form of arg checking to provide feedback to user when illegal configuration is made